leapp/cli/__init__.py

```diff
@@ -40,4 +41,11 @@ def main():
 
     os.environ['LEAPP_HOSTNAME'] = socket.getfqdn()
     _load_commands(cli.command)
-    cli.command.execute('leapp version {}'.format(VERSION))
+    try:
+        cli.command.execute('leapp version {}'.format(VERSION))
+    except UnknownCommandError as e:
+        print(textwrap.dedent('''
+        Command "{}" is unknown.
+        Did you forget to install leapp repositories which provide this command?
```

Comment:
@vinzenz @fernflower thinking about the msg, I think we should change it a little bit as when reading it, I can imagine that some users could feel "weird". Like: ,,How I forget? I installed leapp?!", if you know what I mean. What about something like:

```suggestion
        Hint: Possibly you did not installed particular leapp repositories which provide this command?
              Installing just `leapp`  the system get the framework and utility, not the content that is actually using
              leapp to perform a desired action.
```

Reply:
@pirat89 I really appreciate the move towards improving customer experience! How about we make it super clear what package need to be installed, like "Hint: Possibly you haven't installed the particular leapp repositories (leapp-upgrade-* package) which provide this command?"

Reply:
@fernflower I have been thinking about that, but as this is upstream project and leapp does not have to be used just for the upgrade, I would like to stay with a generic msg. I have been thinking also about something like: "e.g. try to install leapp-{command}", but then we would need to take care about: leapp-answer, leapp-preupgrade, ... it's doable, but not sure we want that.

What about to add something like: E.g. if you are interested about upgrading your system, try to look for leapp-upgrade, etc.

Reply:
Well, I myself as a user would always prefer clear instructions to the generic educational message. 
BTW what do you mean by "leapp does not have to be used just for the upgrade"? leapp won't be used for conversions by convert2rhel, are there any other usages but the upgrade?

Reply:
First, I understand your point on that and I am fighting with myself as I would like to rather write just`"You have not installed the leapp repositories for upgrade. run 'dnf install leapp-upgrade' first"` but I do not see it as the right solution.

It will not be used for conversions for now. The question whether it will be used in future or not has not been decided. This will depend on more stuff in future - e.g. whether some painpoints for other use-cases will be delivered. If we start to focus just on upgrades in this framework and ignore any other possible use-cases (no discussing now just conversions), we end again in situation to work on separation from upgrades if another use-case will be required in the end. If we start to ignore other posibilities, well in such a case we do not need even this PR, as we can just start to require always leapp-upgrade packages and that's it. This discussion we already had couple of times and we agreed to keep project separated and keep it as the upstream project for the framework, that could be used for whatever purposes by anyone. 

I prefer to provide a guidance how the content should be called, so in case that one day the framework is used e.g. for conversions, people will have idea they are searching probably for leapp-convert, etc. This is why we did all renames in packages in the last year.

I think we should stop thinking about leapp as just an upgrade tooling as it's pushing us back, unless we decide it's just the upgrade tooling.

Reply:
btw, discussed with @vinzenz , in case we agree on to provide instructions that will be ok just for rpm/dnf, we could do a simple change in leapp-repository to provide capabilities like `leapp-command(<command>)`, so we could point people to try to install e.g. `leapp-command(preupgrade)`, etc. It's quite simple change. Let's discuss it on cabal.

(P.S. dnf is doing the exactly the same stuff. when the command is not found, it suggests to install `dnf-command(<command>)`)

Reply:
That's a nice idea! However if we'd like to get this PR in before it's implemented I would still add a hint `For example, if you attempt to perform an upgrade, try searching for leapp repositories with dnf --showduplicates list leapp-upgrade*`.
Once the situation changes (leapp gets other usecases or we implement this easy install-leapp-command change, we can change the message).

Reply:
well, the implementation is matter of minutes in this case. it's just one line per cmd in leapp-repository specfile. if we want to go this way, implementation is not problem. but I want to be sure that team is ok to go in this way.